.travis.yml

```diff
@@ -1,167 +1,26 @@
 # Created with package:mono_repo v2.3.0
 language: dart
 
+install:
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then choco upgrade visualstudio2017-workload-vctools -y --force; fi
+  # - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then ./tool/compile_cl.bat; fi
+
 jobs:
   include:
-    - stage: analyze
```

Comment:
Just a heads up, this file is generated using the `mono_repo` package, so you'll need to update the `mono_pkg.yaml` files and re-run the generator instead.

Are you wanting to make this pr to `dart-lang/samples` eventually?

Reply:
I will make a PR but not from this branch. In this branch I'm just testing the windows setup for both msys2 and visual studio ways of compiling the dll.